src/pretix/control/templates/pretixcontrol/order/index.html

```diff
@@ -16,6 +16,9 @@ <h1>
         {% blocktrans trimmed with code=order.code %}
             Order details: {{ code }}
         {% endblocktrans %}
+        {% if order.testmode %}
+            <span class="label label-warning">{% trans "TEST MODE" %}</span>
```

Comment:
Minor cosmetic issue: Make `display: inline-block` to match height and baseline of payment-label opposite.

![screenshot from 2019-02-19 13-08-20](https://user-images.githubusercontent.com/654548/53014010-744c7180-3447-11e9-8fd0-f7e8207b66f5.png)